addons/pharma/functions/fnc_treatmentAdvanced_Painkillers.sqf

```diff
@@ -21,9 +21,10 @@
  */
 
 params ["_medic", "_patient", "_bodyPart", "_classname", "", "_usedItem"];
+private _count1 = [_patient, "Painkillers"] call ACEFUNC(medical_status,getMedicationCount);
```

Comment:
i would rename it personally, count1 is not good name

Reply:
Renamed to painkillerscount